README.md

```diff
@@ -26,15 +26,16 @@ pip install -r requirements.txt
 ```
 
 ### Config file
-Update the information in `config.yaml`.
+Update the information in `config.yaml`. See the [Input args](#input-args) section below for details on the config parameters. 
 
 ### Kubeflow
-Create a Kubeflow deployment using Cloud Marketplace.
-Follow these
+Create a Kubeflow deployment using Cloud Marketplace. Follow these
 [instructions](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
 to give the Kubeflow instance access to GCP services.
 
-## CAIP Demo
+> A future release will automate provisioning of KFP clusters and incorporate K8s Workload Identity for auth. 
+
+## Cloud AI Platform Demo
 This demo uses the scikit-learn model in `examples/sklearn/user_model.py` to
```

Comment:
optional: would you also mind changing `user_model.py` to `sklearn_model.py`. We did the rename a while back and never editted the README. Same applies for the user_model in the KFP section.

Reply:
ack, will do.